content/guides/models/track/log/distributed-training.md

```diff
@@ -101,7 +101,7 @@ The preceding image demonstrates the W&B App UI dashboard. On the sidebar we see
 ### Track all processes to a single run
 
 {{% alert color="secondary"  %}}
-Parameters prefixed by `x_` (such as `x_label`) are in public preview. Create a GitHub issue at [https://github.com/wandb/wandb](https://github.com/wandb/wandb) to provide feedback.
+The new run settings and parameters defined below (mode="shared", x_label, x_primary, etc.) are in public preview. Create a GitHub issue at [https://github.com/wandb/wandb](https://github.com/wandb/wandb) to provide feedback.
```

Comment:
Users won't know what are "new run settings" (potentially won't know what run settings are) so let's be more specific. We don't use latin in the docs (e.g. "etc.")

Suggested change:

```suggestion
The `shared` argument  and parameters prefixed by `x_` (such as `x_label`) described in this section are in public preview. . Create a GitHub issue at [https://github.com/wandb/wandb](https://github.com/wandb/wandb) to provide feedback.
```